### DISCUSSION TITLE
Vuetify 3 Beta - not released?

### DISCUSSION THREAD

**Discussion by sinisarudan** [vuetifyjs/vuetify#14776] [Releases]
according to the https://vuetifyjs.com/en/introduction/roadmap/#v3-0-titan
Vuetify v3.0 (Titan) Beta was planned to be released during February 2022.
Now in March, could we know what is the status? Thanks
P.S. not pushing :)

**Comment by code512** [comment]
I am at the very beginning of new project and I would like to use:
- vue 3 + vite + vuetify 3
- instead vue 2 + vuetify 2 like my old projects....

But, there is still no **v-data-table** component.
I am little bit worried. Any news? (I checked the roadmap)

**Reply by irondog** [reply]
I am in same situation. :(

**Reply by AlvaroBrasilia** [reply]
+1

**Reply by jugurtha114** [reply]
same here

**Comment by joel-wenzel** [comment]
https://github.com/vuetifyjs/vuetify/issues/14729

**Reply by sinisarudan** [reply]
Thanks @joel-wenzel !

**Comment by sinisarudan** [comment]
Cheers! **Beta is officially LIVE!** :)
Beta: [Live](https://next.vuetifyjs.com/)
https://vuetifyjs.com/en/introduction/roadmap/#v3-0-titan
Target Release: May 2022

**Comment by IgorIlgiyaev** [comment]
Yeah there isn't Data Table component at all, Did you removed it?

**Reply by joel-wenzel** [reply]
@IgorIlgiyaev It is still [WIP](https://github.com/vuetifyjs/vuetify/tree/feat/v3-data-table)

**Comment by sinisarudan** [comment]
I have to say that I am also disappointed with how many things are missing even in "vuetify@v3.0.0-beta.1" :(
It would accordingly be very important to know when more precisely the OFFICIAL VUETIFY 3 will be released.
So far it says only "IN MAY"

**Comment by konradgusz** [comment]
At this rate, looking at the list of components it doesn't look like we're gonna get the full release even this year. List of components is very minimal. It doesn't look like beta. Vuetify 2 has around 70 components, some very complex, while Vuetify 3 has 11 basic ones, like sheets, button, divider, image, etc.

Standard definition of a beta version indicates that "beta phase generally begins when the **software is feature complete** but likely to contain a number of known or unknown bugs." I wouldn't say that 11 out of 70 components is "feature complete". Then we gonna have to wait for all the bug fixes and official release candidate.

I'd like to plan few projects ahead and I'm unable to. There's a huge difference for a developer if he has to wait 2-3 months for official release vs 10-nth months because the estimation are bonkers and delivery date keeps moving and I simply cannot use beta versions in production environments.

**Reply by phoenix741** [reply]
I hope too for a release this month.

Do you know a good alternative for vue 3 ?

**Reply by mujahidfa** [reply]
@phoenix741 [Quasar](https://quasar.dev/) is an excellent, high-quality alternative for Material components in Vue 3. I've been using it for Vue 3 projects for the past year and it's been great!

**Comment by sinisarudan** [comment]
Hmm, I was trying to keep bearing with Vuetify 3 (even implementing some workarounds on ongoing projects) until target release is here.
Yet:
- official deadline for Vuetify Target release has passed but we're still in Beta
- there are no answers from developers regarding not-working functionality even the one documented in Beta as working https://github.com/vuetifyjs/vuetify/discussions/15054
- we do not get updates when something is going to be delivered, after the self-defined deadlined has passed, so we can hardly make valid decisions
- as some of you mentioned here, it seems that the TARGET release of 3.0 (expected in the passed May) won't be a "real" target release in sense it will still lack too many components of their functionality, compared to 2.x version ...

I would like to hear some opposing reassuring thoughts from developers, yet it seems that the waiting and workarounds inspired by "too optimistic" announcements by Vuetify developers is not justified anymore :(
Especially that this is not ony waiting for Vuetify 3 but also preventing us from switching from Vue2 to Vue 3 and all the new ecosystem being built around it

Accordingly, I would also switch the above-mentioned [Quasar](https://quasar.dev/) and if not too late, might switch back to Vuetify 3 when it reaches that level of stability and versatility.

I checked earlier  [Quasar](https://quasar.dev/) but didn't want to bring it here as being an alternative, until @mujahidfa mentioned it and until I had patience to wait for some progress here :)

**Comment by konradgusz** [comment]
At the beginning of my project vuetify seemed like a great framework to use, but current approach to communicating releases, deadlines and delays makes it a risky option at best.

Moving to a whole new framework will be a lot of work, but I just can't risk it anymore to be stuck with outdated project. 
I'll be recommending my team to drop vuetify support and investigate another framework of choice with Quasar looking like a very strong candidate.

I think if there was a clear way of using vuetify2 with vue3/nuxt3 projects it would solve all our problems and we could all happily wait for new and better version of vuetify.

**Reply by sinisarudan** [reply]
I'm getting the similar feeling and thoughts ...

**Reply by mgfouad** [reply]
hopefully to jump on v3 soon, it is a critical decision to rely on a beta version

**Comment by ThaDaVos** [comment]
To everyone here, please see https://github.com/vuetifyjs/vuetify/issues/14984 - for the up to date state - a lot of components are in the latest Beta, some are moved to V3.1 due technical and refactoring reasons as far as I could find - don't forget, it was a complete rewrite because the whole render functions stuff changed and they used it a lot!
